README.md

```diff
@@ -1,85 +1,100 @@
 
-Down below there is an example you can use (**(For Local test / regtest only)**)
+To check that it worked, open a new terminal instance and run `java -version` command. It should output something like 
 
 ```
-federator {
-    enabled = true
-    
-    signers {
-       BTC {
-          type = "keyFile"
-          path = "A/PATH/TO/YOUR/BTC-KEY.key" # This should point to your generated key file
-       }
-       RSK {
-          type = "keyFile"
-          path = "A/PATH/TO/YOUR/RSK-KEY.key" # This should point to your generated key file
-       }
-       MST {
-          type = "keyFile"
-          path = "A/PATH/TO/YOUR/MST-KEY.key" # This should point to your generated key file
-       }
-    }
-    # peers for the bitcoin network
-    bitcoinPeerAddresses = [
-        "127.0.0.1:18444" #bitcoind p2p port.
-    ]
-}
+openjdk version "1.8.0_352"
+OpenJDK Runtime Environment
+OpenJDK 64-Bit Server VM 
 ```
-**Note: You can use the same key file for BTC, RSK, and MST**
 
-**Important:**
-When setting up the Bitcoind Node host:port for the powpeg-node you MUST use the p2p port.
-The powpeg-node will “connect” even if you setup the RPC port, but the connection won’t go through and the powpeg-node won’t start.
+*If not, try closing the terminal and open a new one cause sometimes it stays cached.*
+
+### Bitcoind
 
-### Fetch the code!
+To install version `0.18.0`
 
 ```bash
-git clone https://github.com/rsksmart/powpeg-node
-cd powpeg-node
+curl -O https://bitcoin.org/bin/bitcoin-core-0.18.0/bitcoin-0.18.0-osx64.tar.gz
+tar -zxf bitcoin-0.18.0-osx64.tar.gz
+sudo mkdir -p /usr/local/bin
+sudo cp bitcoin-0.18.0/bin/bitcoin* /usr/local/bin/.
+rm -rf bitcoin-0.18.0
 ```
 
-### Verify the code!
+To validate it run `bitcoind -daemon`.
+
+It should output ```Bitcoin server starting```
+
+Run `bitcoin-cli stop` afterwards.
+
+
+
+## Setting up the project
+Create a directory (for example, “powpeg-project”) to hold the rskj node, the powpeg node and further configurations.
+
+### Fetch the code
+
+Inside the `powpeg-project` directory clone the powpeg-node repository.
+
+```bash
+git clone https://github.com/rsksmart/powpeg-node
+```
 
+```bash
+cd powpeg-node
+```
 
-Before anything, you must ensure the security chain of the source code. For that, you must go through the following steps. For Linux based OS (Ubuntu for example) it's recommended install `gnupg-curl` to download the key through HTTPS.
+### Verify the code
 
+Before anything, you must ensure the security chain of the source code. For that, you must go through the following steps. For Linux based OS (Ubuntu for example) it’s recommended install `gnupg-curl` to download the key through HTTPS.
 
 1. Download sec channel public key
 
 ```bash
-$ $ gpg --keyserver https://secchannel.rsk.co/SUPPORT.asc --recv-keys A6DBEAC640C5A14B
+gpg --keyserver https://secchannel.rsk.co/SUPPORT.asc --recv-keys A6DBEAC640C5A14B
```

Comment:
This command does not work for me. Is working for you?

```bash
> gpg --keyserver https://secchannel.rsk.co/SUPPORT.asc --recv-keys A6DBEAC640C5A14B
gpg: keyserver receive failed: Network is unreachable
```

Reply:
you are right, it doesn't work for us either right now. we will try to understand why and solve it